ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/RoundKudf.java

```diff
@@ -20,11 +20,27 @@
 
 public class RoundKudf implements Kudf {
 
+  public static final String NAME = "ROUND";
```

Comment:
This class would benefit from a unit test. The JSON test is a functional test to ensure everything is wired up correctly, but a unit test is still a good thing and can test things the functional test can't, e.g. that it throws on the wrong params count / type.

Reply:
https://github.com/confluentinc/ksql/pull/3028/commits/d04b758a72b748f35a5185b6ca8e97da61daf15e